src/tools/U8U16Test/U8U16Test.hpp

```diff
@@ -1,34 +1,23 @@
-/*++
-Copyright (c) Microsoft Corporation
-Licensed under the MIT license.
-
-Module Name:
-- UTF8OutPipeReader.hpp
-
-Abstract:
-- This reads a UTF-8 stream and gives back a buffer that contains complete code points only
-- Partial UTF-8 code points at the end of the buffer read are cached and prepended to the next chunk read
-
-Author(s):
-- Steffen Illhardt (german-one) 12-July-2019
---*/
+﻿// TEST TOOL U8U16Test

```

Comment:
Hey could we maybe elaborate more on what this file is used for? I was certainly confused when reading the code that there were not only different implementations for the conversion within this file, but also that it wasn't using code from the actual `til` implementation. I get now that it's being used to figure out the perf of various different implementations, and that `til` actually only implements the fastest versions from this project, but I was pretty confused.

Reply:
Yes. I will add some more comments to clarify what the files are for.